WordPress/gutenberg
# Audio block: test the block for visual regressions

## What?


This PR introduces screenshots to test for visual regressions on blocks. Hopefully it can help testing changes like https://github.com/WordPress/gutenberg/pull/57841 against regressions. This is a proof of concept and right now it's only testing the audio block, but if we agree this is a good thing to have we can extend to other blocks and variations of them (or even test against different default themes)

## Why?


Visual regressions are tricky to prevent some times, having another tool (automated even better) should help with confidence checks on our changes.

## How?


Every time we run the test, we compare against the screenshot that the test uses as a snapshot. If there are any visual changes, the test will fail.

## Testing Instructions





Run `npm run test:e2e:playwright -- -g audio.spec.js` on a first run this should pass 
Now make a change to `packages/block-library/src/audio/style.scss` like:

```
.wp-block-audio {
	background: red;
}
```

Run the test again, it should fail now.
